akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -272,11 +272,11 @@ final case class RootActorPath(address: Address, name: String = "/") extends Act
 
   override val toSerializationFormat: String = toString
 
-  override def toStringWithAddress(addr: Address): String =
+  override def toStringWithAddress(newAddress: Address): String =
```

Comment:
Tests as well as the testkit (which is public API but exempt from these rules) are fine to change :)